packages/components/src/components/Table/Table.js

```diff
@@ -0,0 +1,173 @@
+        render={({
+          rows,
+          headers,
+          getHeaderProps,
+          getRowProps,
+          getSelectionProps,
+          getBatchActionProps,
+          selectedRows
+        }) => (
+          <TableContainer title={showTitle ? resource : null}>
+            <TableToolbar>
+              <TableBatchActions {...getBatchActionProps()}>
+                {batchActionButtons.map(button => (
+                  <TableBatchAction
+                    renderIcon={button.icon}
+                    key={`${button.text}Button`}
+                    onClick={() => {
+                      button.handler(
+                        selectedRows,
+                        getBatchActionProps().onCancel
+                      );
+                    }}
+                  >
+                    {button.text}
+                  </TableBatchAction>
+                ))}
+              </TableBatchActions>
+              <TableToolbarContent>
+                {toolbarButtons.map(button => (
+                  <Button
+                    disabled={loading}
+                    onClick={button.handler}
+                    renderIcon={button.icon}
+                    key={`${button.text}Button`}
+                  >
+                    {button.text}
+                  </Button>
+                ))}
+              </TableToolbarContent>
+            </TableToolbar>
+            {loading ? (
+              <DataTableSkeleton
+                useZebraStyles
+                rowCount={1}
+                columnCount={headers.length}
+              />
+            ) : (
+              <CarbonTable>
+                <TableHead>
+                  <TableRow>
+                    {batchActionButtons.length > 0 && (
+                      <TableSelectAll {...getSelectionProps()} />
+                    )}
+                    {headers.map(header => {
+                      return (
+                        <TableHeader
+                          {...getHeaderProps({ header })}
+                          className="cellText"
+                          key={header.key}
+                        >
+                          {header.header}
+                        </TableHeader>
+                      );
+                    })}
+                  </TableRow>
+                </TableHead>
+                <TableBody>
+                  {rows.map(row => {
+                    return (
+                      <TableRow
+                        {...getRowProps({ row })}
+                        key={row.id}
+                        className="row"
+                      >
+                        {batchActionButtons.length > 0 && (
+                          <TableSelectRow {...getSelectionProps({ row })} />
+                        )}
+                        {row.cells.map(cell => (
+                          <TableCell
+                            key={cell.id}
+                            id={cell.id}
+                            className="cellText"
```

Comment:
can we target existing Carbon class names or do we really need to add our own here?

Reply:
ill double check this.